zipline/utils/api_utils.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2015 Quantopian, Inc.
```

Comment:
(Agreed they don't belong with api_support.py - but the names being so similar I find confusing.)